packages/navigator/package.json

```diff
@@ -70,7 +68,8 @@
     "typescript": "~3.9.0",
     "url-loader": "~3.0.0",
     "watch": "~1.0.2",
-    "webpack": "5.0.0-beta.21",
+    "webpack": "^5.0.0",
```

Comment:
We can probably stick to `"webpack": "^4.0.0"` for now so it uses the same version as in lab 2

Reply:
And then bump to `^5.0.0` when updating to lab 3.

Reply:
Indeed I agree, it will be nice to release first in the current Jupyter environment. Then switch to JLab3.